nav2_bringup/spawn_turtlebot3/src/spawn_turtlebot3/spawn_turtlebot.py

```diff
@@ -0,0 +1,77 @@
+"""
+spawn_turtlebot.py
+
+Script used to spawn a turtlebot in a generic position
+"""
+import os
+import sys
+import rclpy
+from ament_index_python.packages import get_package_share_directory
+from gazebo_msgs.srv import SpawnEntity
+import xml.etree.ElementTree as ET
+
+
+def main():
+    """ Main for spwaning turtlebot node """
+    # Get input arguments from user
+    argv = sys.argv[1:]
+
+    # Start node
+    rclpy.init()
+    node = rclpy.create_node("entity_spawner")
+
+    node.get_logger().info(
+        'Creating Service client to connect to `/spawn_entity`')
+    client = node.create_client(SpawnEntity, "/spawn_entity")
+
+    node.get_logger().info("Connecting to `/spawn_entity` service...")
+    if not client.service_is_ready():
+        client.wait_for_service()
+        node.get_logger().info("...connected!")
+
+    # Get path to the turtlebot3 burgerbot
+    sdf_file_path = os.path.join(
+        get_package_share_directory("turtlebot3_gazebo"), "models",
+        "turtlebot3_burger", "model.sdf")
```

Comment:
Can you only spawn the burger model and not the waffle?

Reply:
Good catch